native_app/README.md

```diff
@@ -0,0 +1,71 @@
+# Native compilation sample
+
+This sample is a command line application that can be compiled to native code
+using the `dart2native` command included in Dart 2.6. The application fetches
+stats for a GitHub user and prints them to the console.
+
+This sample also shows how to parse command line options into Dart object using
+[`package:build_cli_annotations`][build-cli] and [`package:build`][build].
```

Comment:
Is the additional complexity incurred by adding `build_cli_annotations` rather than just `args` worth it for a simple example like this?

Reply:
I think you're right, this is probably out of scope. We could refactor this sample into a package:build sample and make the dart2native sample simpler. I created #17 to track.